### PR TITLE
Bumped retry-axios

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3045,10 +3045,9 @@ ret@~0.1.10:
   version "0.1.15"
   resolved "https://registry.yarnpkg.com/ret/-/ret-0.1.15.tgz#b8a4825d5bdb1fc3f6f53c2bc33f81388681c7bc"
 
-retry-axios@^0.4.1:
-  version "0.4.1"
-  resolved "https://registry.yarnpkg.com/retry-axios/-/retry-axios-0.4.1.tgz#22ee392c6f20ae858d06650fa06091ea4f3406e5"
-  integrity sha512-h3mdzDUw4MlvzjxJ96mysapyxcHzAmGVywrBtU5oAtXI2aBxMEgcmyepkKfoVXK6we2padRl1BwwvL1N7+lPwA==
+"retry-axios@git://github.com/f1cognite/retry-axios.git#7ea9b893cf730b4f75743f623a6d321ed7699acf":
+  version "0.3.2"
```
Comment (valgussev) [inline yarn.lock:3049]: Just curious why the version is different, have you forked an older one?

**Review by valgussev — APPROVED** [review]
Please don't forget to bump package.json version

File: yarn.lock (hunk 1 above)
Comment (ghost) [reply yarn.lock:3049]: I see that on npm it is `0.4.1` but the version on their master branch is `0.3.2`: https://github.com/JustinBeckwith/retry-axios/blob/master/package.json

Comment (ghost) [reply yarn.lock:3049]: It passes our retry-test in the SDK so it works.

**Pull request merged**